SingularityUI/app/models/Request.coffee

```diff
@@ -152,7 +157,16 @@ class Request extends Model
                     return false
 
                 else
-                    localStorage.setItem(@localStorageCommandLineInputKeyPrefix + @id, commandLineInput) if commandLineInput?
+                    history = localStorage.getItem(@localStorageCommandLineInputKeyPrefix + @id)
```

Comment:
it would be nice if we could move all this local storage related stuff out to some utility file. it always bugged me that we already have all this vex stuff in a model file, maybe we can tackle that in another issue, but it would be nice if we could lean this out a bit.